app/models/solr_document.rb

```diff
@@ -28,4 +26,12 @@ class SolrDocument
 
   # Do content negotiation for AF models.
   use_extension(Hydra::ContentNegotiation)
+
+  # overrides the MimeTypes included by CurationConcerns::SolrDocumentBehavior in order to add psd types.
+
+  module Hydra::Works::MimeTypes::ClassMethods
```

Comment:
I'm overriding a class method in the Hydra Works Mime Types module, which is included in our project by CurationConcerns SolrDocumentBehaviors, so I don't understand where else I should be doing it.

https://github.com/projecthydra/hydra-works/blob/d41be82540eca9d3a76301df3dcea5b74c9d5937/lib/hydra/works/models/concerns/file_set/mime_types.rb#L28

I can write a unit test about this method for the solr document, definitely, but this test relies on the new psd mime type and would fail without it:

https://github.com/aic-collections/aicdams-lakeshore/blob/jp2-mogrify/spec/lib/hydra/derivatives/processors/image_spec.rb#L14-L25